sample_size/multiple_testing.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+import numpy as np
+from statsmodels.stats.multitest import multipletests
+
+from sample_size.metrics import BaseMetric
+
+
+class MultipleTestingMixin:
+    """
+    This class calculates sample size required under the case of multiple testing
+
+    Attributes:
+    metrics: a list of BaseMetric registered by users
+    variants: number of variants, including control
+    alpha: statistical significance
+    power: statistical power
+
+    """
+
+    metrics: List[BaseMetric]
+    alpha: float
+    power: float
+    variants: int
+
+    REPLICATION: int = 100
+
+    def get_multiple_sample_size(self, lower: float, upper: float, depth: int = 0) -> int:
+        max_recursion_depth: int = 20
+        epsilon: float = 0.025  # TODO(any): make this configurable by users ML-5429
```

Comment:
I didn't put them as default attributes since they are used only in this method. 

Yes, I just created it: https://github.com/godaddy/sample-size/pull/35